substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -523,7 +533,7 @@ pub mod pallet {
 		/// This is useful for out-of-band promotions, hence it has its own `FastPromoteOrigin` to
 		/// be (possibly) more restrictive than `PromoteOrigin`. Note that the member must already
 		/// be inducted.
-		#[pallet::weight(T::WeightInfo::promote_fast(*to_rank as u32))]
+		#[pallet::weight(T::WeightInfo::promote_fast(*to_rank))]
```

Comment:
Would need to run benchmarks for this 🙏

Reply:
Why? Generally we have now these weight updates once a week, that should be fine to be done as part of them.